src/cursesio.py

```diff
@@ -0,0 +1,112 @@
+import curses
+from tiles import cross, circle, empty
+
+OFFSET_X = 5
+
+DIRECTIONS = ['KEY_LEFT', 'KEY_RIGHT']
+
+
+def clear(stdscr):
+    stdscr.clear()
+
+
+def get_player(number):
+    if number == 1:
+        return '1'
+    if number == -1:
+        return '2'
+    return None
+
+
+def get_symbol(value):
+    if value < 0:
+        return 'O'
+    if value > 0:
+        return 'X'
+    return ' '
+
+
+def print_winner(stdscr, winner):
+    stdscr.addstr(f'\n\nThe player {get_player(winner)} has aligned 5 symbols. Game Won !', curses.A_BOLD)
+    stdscr.addstr('\nPress any key to quit !')
+    stdscr.getkey()
+
+
+def print_player(stdscr, player, phase=1):
+    stdscr.addstr(f'\n\nPlayer {get_player(player)} ({get_symbol(player)})')
+    text = 'Press space to select a '
+    text += 'cube to move' if phase == 1 else 'destination for the cube'
+    stdscr.addstr(f'\n{text}')
+
+
+def get_attr(pos, movables, selected, picked):
```

Comment:
This function name isn't very descriptive.
What is the attribute does is return?

Reply:
Renamed get_style